### ISSUE TITLE
a11y: keyboard navigation on video challenges

### ISSUE THREAD

**Issue by ojeytonwilliams** [freeCodeCamp/freeCodeCamp#39124]
**Describe the bug**
There is no visual indication that you have reached the radio buttons or the 'Check your answer' button via tab navigation.

**To Reproduce**
1. Go to https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/introduction-why-program
2. Tab through the page
3. See the embedded video and links get outlined
4. Once you've reached 'Install Python on MacOS' press tab again
5. See that nothing is outlined

**Expected behavior**
The radio button group should be outlined.  The current selection is indicted by the dot, but not that you have selected the buttons.  The 'Check your answer' button also should get outlined when you reach it.

**Screenshots**
![image](https://user-images.githubusercontent.com/15801806/85519950-8bced800-b602-11ea-8576-1f0dbda5ef67.png)

**Comment by Sky020** [comment]
I believe this has come about, because we hid the default input styles with the class: `video-quiz-input-hidden`
Giving the `label` a tabindex of `0` does the trick. The button needs this style changed:
```
.btn-primary.focus, .btn-primary:focus, .btn:focus {
    background-color: var(--quaternary-background);
    color: var(--secondary-color);
    border-color: var(--secondary-color);
```

**Comment by ojeytonwilliams** [comment]
Ah, thanks for taking a look.  Could you pop the changes in a PR?  It should be nice and easy to review.

**Comment by moT01** [comment]
Side note: I believe there's a similar problem with pretty much all the challenges. [The first challenge](https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/say-hello-to-html-elements): Can't reach the buttons with tab. Same with the [project pages](https://www.freecodecamp.org/learn/responsive-web-design/responsive-web-design-projects/build-a-tribute-page): Can't reach the input or button with tabs.

**Comment by ojeytonwilliams** [comment]
@moT01 is it getting stuck in the editor?  Unless you manually put it into accessibility mode, the editor automatically gets focus when you start a challenge and you have to Escape out.

**Comment by moT01** [comment]
My bad, it looks like it does make it there. I thought it was skipping over them.

**Comment by ojeytonwilliams** [comment]
It does get stuck in the editor if you keep pressing tab, though.  Ideally it should tab onto the editor and then, if you want to use it, you press enter.  If you want to move on to console or preview, tab should do that for you.

**Comment by raisedadead** [comment]
Hi, 

We are closing old issues/pull-requests that we think are stale. If you need still need help with this or are fixing this, please open a fresh issue/pull-request.

Thanks for your understanding. 

Happy contributing.
